frame/utility/src/tests.rs

```diff
@@ -25,6 +25,7 @@ use frame_support::{
 	assert_ok, assert_noop, impl_outer_origin, parameter_types, impl_outer_dispatch,
 	weights::Weight, impl_outer_event, dispatch::DispatchError, traits::Filter, storage,
 };
+use frame_system as system;
```

Comment:
Also here.

Reply:
Same to the reason in `frame/sudo/src/mock.rs`.

Mixed `frame_system` and `system` also happens here, will update.